src/main/java/org/folio/inventory/domain/items/LastCheckIn.java

```diff
@@ -0,0 +1,137 @@
+
+package org.folio.inventory.domain.items;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.vertx.core.json.Json;
+import io.vertx.core.json.JsonObject;
+import java.util.Date;
+import javax.validation.constraints.Pattern;
+import org.joda.time.DateTime;
+import org.joda.time.format.ISODateTimeFormat;
+
+/**
+ * Information about when an item was last scanned in the Inventory app.
+ *
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder( {
+    "dateTime",
+    "servicePointId",
+    "staffMemberId"
+})
+public class LastCheckIn {
+
+    static final LastCheckIn EMPTY = new LastCheckIn();
```

Comment:
Can we just use `null` instead of an `EMPTY` object? I don't see what this `EMPTY` object gives us over `null` here. Also, this class is mutable, so the `EMPTY` object could be modified, which doesn't seem right if this is to be a constant.

Reply:
Sure.